### PR TITLE
[13.0][MIG] connector and test_connector

### PR DESCRIPTION
**Pull request by Laurent-Corron**
- [x] requires #358

### CODE REVIEW

**Review by lmignon — CHANGES_REQUESTED** [review]
@Laurent-Corron To be complete, this PR should also include the migration of the addon used to test the connector addon https://github.com/OCA/connector/tree/12.0/test_connector

**Comment by Laurent-Corron** [comment]
@lmignon 
test_connector added :)

**Comment by sebalix** [comment]
@Laurent-Corron it now requires #358 instead of #355.

**Comment by lmignon** [comment]
rebased and fixed pre-commit...

**Review by lmignon — APPROVED** [review]

**Review by simahawk — APPROVED** [review]
LG.

**Review by sebalix — APPROVED** [review]

**Comment by guewen** [comment]
#358 is merged!

**Review by guewen — APPROVED** [review]
Thanks :)

File: connector/__manifest__.py [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-          ],
- 'installable': False,
- }
+{
+    "name": "Connector",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp,Openerp Connector Core Editors,"
+    "Odoo Community Association (OCA)",
+    "website": "http://odoo-connector.com",
+    "license": "LGPL-3",
```
Comment (mlaitinen) [inline connector/__manifest__.py:10]: So ... `connector` is LGPL-3 while `queue_job` (dependency) is AGPL-3? Is this legally possible?

Comment (guewen) [reply connector/__manifest__.py:10]: You have a point. The legal question is hard to answer to me.
In any case, I think we should change `queue_job`'s license to LGPL-3 for the same reason as mentioned on #356

Comment (guewen) [reply connector/__manifest__.py]: Here's the PR for changing the license https://github.com/OCA/queue/pull/184

Comment (guewen) [reply connector/__manifest__.py:10]: Here's the PR for changing the license https://github.com/OCA/queue/pull/184

**Comment by lmignon** [comment]
rebased and fixed isort error after rebase...

**Pull request merged**